### ISSUE TITLE
File Downloads not working

### ISSUE THREAD

**Issue by alexbgameson** [lukegackle/WebViewTools#4]
Hello!

File downloads and uploads do not work with this extension.

Is this a bug?

**Comment by TIMAI2** [comment]
Agreed. It would be great of you could get the input = "file" to open a file picker (even better the camera)
